view/frontend/web/template/payment/hpp-form.html

```diff
@@ -23,9 +23,9 @@
 -->
 
 <!-- TODO check if it can be outsources to an adyen-methods template to be used by all the methods-->
```

Comment:
@tnaber I think you can also get rid of the above TODO line since this is now different for each PM